sql/item_strfunc.cc

```diff
@@ -1449,6 +1449,72 @@ String *Item_func_sformat::val_str(String *res)
   return null_value ? NULL : res;
 }
 
+#include"my_global.h"
+#include <openssl/rand.h>
+#include <openssl/err.h>
```

Comment:
can we avoid these includes by using `my_random_bytes` directly instead of `RAND_bytes`?